code/index.html

```diff
@@ -3,28 +3,105 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>A.F.R.</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="icon" type="image/png" href="https://img.icons8.com/android/24/000000/clenched-fist.png"/>
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
+    <header>
+      <div class="hero-image"><img src="./images/banner.jpg"/>
+        <div class="hero-text">
+          <h1 class="rotate">Express</h1>
+          <h1>yourself</h1></div></div>
+          <div class="small-text">
+          <h2>art</h2><br>
+          <h2>freedom</h2><br>
+          <h2>resistance</h2>
+        </div>
+      </header>
+   
+    <section class="container">
+      <form class="form" action="http://httpbin.org/anything" method="POST">
+        <h3>Order a sample t-shirt for free</h3>
+      <div class="form-group">
+        <label for="Name"><h4>Full name</h4></label>
+          <input class="input-text" type="text" id="name" name="name" placeholder="Name..." required><br>
+        </div>
+      <div class="form-group">
+        <label for="Name"><h4>Country</h4></label>
+          <input class="input-text" type="text" id="country" name="country" placeholder="Country..." required><br>
+        </div>
+      <div class="form-group">
+        <label for="Name"><h4>Address and zip code</h4></label>
+          <input class="input-text" type="text" id="address" name="address" placeholder="Address..." required><br>
+        </div>
+      <div class="form-group">
+        <label for="Name"><h4>E-mail address</h4></label>
+          <input class="input-text" type="email" id="email-address" name="e-mail address" placeholder="e-mail..." required><br>
```

Comment:
To connect you label with 'for' to the input you need to use id, some reading here:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

Also try to stay away from `<br>` since you should try to solve this by using some CSS styling instead ☕️